features/providers/check_multiple_employment.feature

```diff
@@ -146,7 +146,7 @@ Feature: Check multiple employment
     When I click 'Save and continue'
     Then I should be on a page showing "Confirm the following"
 
-    When I check "I confirm the above is correct and that I'll obtain a signed declaration from my client."
+    When I select the first checkbox
```

Comment:
be good to have a more descriptive step here, like the old one. Have tried and this works

```suggestion
    When I check "I confirm the above is correct and that I'll get a signed declaration from my client."
```

Reply:
The reason I changed these was because it didn't feel like the specific content was important.

But I can change these back if we prefer 👍

Reply:
@cpjmcquillan for me the use of "I select the first checkbox" is not a pattern/style I like for reasons mentioned and would change elsewhere too if found. 

can we change this and get this in?